src/main/java/uk/gov/ons/ctp/integration/event/generator/NativeRabbitEventGeneratorExample.java

```diff
@@ -19,7 +21,7 @@ public static void main(String[] args) {
     try {
       connectionDetails = new RabbitConnectionDetails();
```

Comment:
?

Reply:
I think that this is ok where it is. Its naming makes it look like it's in the wrong place.

It makes use of the EventGenerator so really has to live in this project. It's also using the NativeRabbitEventSender hence the use of the connection details object.

Reply:
After discussion, I'm deleting this class.